pkg/controller/v1beta1/inferenceservice/reconcilers/deployment/deployment_reconciler.go

```diff
@@ -77,6 +77,9 @@ func createRawDeployment(componentMeta metav1.ObjectMeta,
 			},
 		},
 	}
+	if componentExt.DeploymentStrategy != nil {
+		deployment.Spec.Strategy = *componentExt.DeploymentStrategy
```

Comment:
From my understanding, rawDeployment is using HPA as a autoscaler. This means that HPA will overwrite deployment replicas. So I guess hpa and this controller will continuously try to update this deployment, what do you think?

Reply:
It should be fine since HPA will only update the replicas field. Here we are only modifying the deployment strategy so it won't be overwritten.

Reply:
Ok it only changed strategy so this source is ok.